deploy/helm/kubernetes/templates/controller-manager-deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   labels:
     app: "{{ $fullName }}-controller-manager"
 spec:
-  replicas: 1
+  replicas: {{ .Values.controller_manager.replicas }}
```

Comment:
Commonly helm chars, the same as go are not using underscore (`_`) in parameters, thus `controllerManager` instead of `controller_manager` is more corrrect.